### ISSUE TITLE
Alt-text of informational icon - Disclaimer List on Main Page

### ISSUE THREAD

**Issue by angelina-smith** [GSA/px-benefit-finder#1398]
### What's expected

Location: Benefit-finder main page
What's expected: Informational 'i' icon is given a role of image, and aria-label is where the text label should be.
Screen reader should announce: the 'i' icon as important, then read the following text

### What happened

Screen reader is currently not reading the 'i' icon as important.

### How to replicate

NVDA x Chrome on Dev site.

**Comment by angelina-smith** [comment]
@scottqueen-bixal Let me know if I wrote up the expected behavior of role=image correctly. If not, I can def fix. Tech lingo is not my strong suit.
